packages/gemini-image/CONTRIBUTING.md

```
@@ -0,0 +1,170 @@
+        ValueError: If model_key is invalid.
+        APIError: If the Gemini API returns an error.
+
+    """
+```
+
+## Pull Request Process
+
+1. **Create a feature branch** from `main`
+2. **Make your changes** with tests
+3. **Run quality checks**:
+   ```bash
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)